docs/overlay.md

```diff
@@ -99,22 +99,22 @@ You can also setup networks and services and then attach a running container to
 **host-1**:
 
 ```
-docker network create -d overlay prod 
-docker network ls
-docker network info prod
-docker service publish db1.prod
-cid=$(docker run -itd -p 8000:8000 ubuntu)
-docker service attach $cid db1.prod
```

Comment:
it was done this way for ease of copy + paste for those who like to give a quick try :-)
do you still see a need to add the term prompt `$` ?